src/lsm/README.md

```diff
@@ -50,7 +50,7 @@ The immutable table is compacted into level 0 during the odd level half of the b
 At any given point, there are at most `⌈levels/2⌉` compactions running concurrently.
 The source level is denoted as `level_a` and the target level as `level_b`.
 The last level in the LSM tree has no target level so it is never a source level.
-Each compaction compacts a [single table](#table-selection) from `level_a` into all tables in
+Each compaction compacts a single table from `level_a` into all tables in
```

Comment:
@sentientwaffle maybe this one is worth calling out since there is no section on this page that I could figure out that was relevant to be linked here. The original link doesn't work.

Reply:
It should link to `#compaction-selection-policy`.

Reply:
kk thanks